packages/persistent-ts/src/Vector.ts

```diff
@@ -88,7 +89,7 @@ export class PersistentVector<T> implements Iterable<T> {
   get(index: number): T | undefined {
     if (index < 0 || index >= this.length) return undefined;
 
-    let array;
+    let array: T[] | (INode<T> | undefined)[];
```

Comment:
very strange type

Reply:
Yes but it's what the type of `this.tail` which was assigned implicitly earlier.